setup.py

```diff
@@ -67,7 +67,13 @@
 
 install_requires = [
     'Flask>=0.11.1',
-    'requests>=2.4.0',
+    # elasticsearch has hard version range dependency on requests and urllib3
+    # requests has hard version range dependency on idna
+    # Every time idna and urllib3 are updated, installation breaks because
+    # elasticsearch and requests dependencies are not resolved properly.
+    'requests>=2.4.0,<3.0.0', # from elasticsearch
```

Comment:
This seems to be a [`test_requires` dependency of `elasticsearch-py` (for all of its versions)](https://github.com/elastic/elasticsearch-py/blob/3d0bb532f64e1c032e5f76c68026f5357c9ce730/setup.py#L15).